daq/topology.py

```diff
@@ -131,8 +131,7 @@ def get_device_intfs(self):
             device_intfs.append(intf_names[port-1] if named_port else default_name)
         return device_intfs
 
-    def direct_device_traffic(self, device, port_set):
-        """Modify gateway set's vlan to match triggering vlan"""
+    def _populate_set_devices(device, port_set):
```

Comment:
is this missing "self"

Reply:
Yup. Wasn't complete yet. Was just a draft to run the changes by